editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -163,29 +169,37 @@ export function SectionsOverviewList({
 
 const Grid = styled("dl")(({ theme }) => ({
   display: "grid",
-  gridTemplateColumns: "1fr 200px",
+  gridTemplateColumns: "1fr 220px",
   gridRowGap: "5px",
-  marginTop: theme.spacing(2),
-  marginBottom: theme.spacing(2),
+  paddingTop: theme.spacing(3),
+  paddingBottom: theme.spacing(2),
   "& > *": {
-    borderBottom: "1px solid grey",
-    paddingBottom: theme.spacing(2),
-    paddingTop: theme.spacing(2),
+    borderBottom: `1px solid ${theme.palette.secondary.main}`,
+    paddingBottom: theme.spacing(1.5),
+    paddingTop: theme.spacing(1.5),
     verticalAlign: "top",
     margin: 0,
   },
+  "& > dt:first-of-type, & > dd:first-of-type": {
+    borderTop: `1px solid ${theme.palette.secondary.main}`,
+  },
```

Comment:
Is this top border for the first list item GovUK standard? 

It's the only Sections change here that stands out as a bit odd to me! Mostly because it feels inconsistent with how Checklists & the Review page use borders between list items (eg only bottom).
![Screenshot from 2023-06-13 17-31-19](https://github.com/theopensystemslab/planx-new/assets/5132349/4f11cfe8-983b-48ec-a541-7a57fc3ad332)
![Screenshot from 2023-06-13 17-32-16](https://github.com/theopensystemslab/planx-new/assets/5132349/9b8d82b4-218c-4d46-90bc-b04f179a524d)
![Screenshot from 2023-06-13 17-34-13](https://github.com/theopensystemslab/planx-new/assets/5132349/85998f3f-a05e-4aad-8c89-44a791c5419a)

Reply:
GOV.UK are also inconsistent with this (you caught them, HA!). Their task list (our sections) has borders top & bottom and summary list (our review) has only bottom. They also do accordions very differently so we can disregard that.

For simplicity I'll remove the top border so that _our_ components are consistent.

Reply:
Thanks for looking into this one & adjusting !